src/battle.ts

```diff
@@ -34,6 +34,8 @@ type WeatherState = [string, number, number];
 type EffectTable = {[effectid: string]: EffectState};
 type HPColor = 'r' | 'y' | 'g';
 
+const PLAYER_GROUP = '\u2606';
```

Comment:
How does the ignoreSpects code work usually? Presumably you only want to ignore unauthed users and maybe voices.

Reply:
Yes, exactly -- but the logic for displaying `|j|` and `|l|` messages didnt respect that.